tdp/core/deployment/deployment_runner.py

```diff
@@ -66,9 +66,11 @@ def _run_operation(self, operation_rec: OperationModel) -> None:
             return
 
         # Execute the operation
-        playbook_file = self._collections[operation.collection_name].playbooks[
-            operation.name
-        ]
+        playbook_file = self._collections[
+            operation.collection_name
+        ].playbooks_directory / (
+            operation.name + ".yml"
```

Comment:
Why this modification ? This is bad, we had the path to the playbook file without this PR.